### PR TITLE
[DOCS] Update docstrings for  GPT2 and  Whisper tokenizer

### PR DESCRIPTION
**Pull request by McDonnellJoseph**
# What does this PR do?

- [x ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Comment by McDonnellJoseph** [comment]
@ydshieh Does this look good for you?

File: src/transformers/models/gpt2/tokenization_gpt2_fast.py [hunk 1]
```diff
@@ -95,25 +95,23 @@ class GPT2TokenizerFast(PreTrainedTokenizerFast):
     refer to this superclass for more information regarding those methods.
 
     Args:
-        vocab_file (`str`):
+        vocab_file (`str`, *optional*):
```
Comment (ydshieh) [inline src/transformers/models/gpt2/tokenization_gpt2_fast.py:98]: @ArthurZucker 

 I think `vocab_file` is required in order to create a tokenizer, despite we have a default `None` value. Having `*optional*` will create confusion. Not sure what we can do here (for the script)

File: src/transformers/models/whisper/tokenization_whisper_fast.py [hunk 2]
```diff
@@ -95,28 +95,26 @@ class WhisperTokenizerFast(PreTrainedTokenizerFast):
...
-        normalizer_file (`str`, *optional*, defaults to `None`):
+        normalizer_file (`str`, *optional*):
             Path to the normalizer_file file.
-        errors (`str`, *optional*, defaults to `"replace"`):
-            Paradigm to follow when decoding bytes to UTF-8. See
-            [bytes.decode](https://docs.python.org/3/library/stdtypes.html#bytes.decode) for more information.
-        unk_token (`str`, *optional*, defaults to `<|endoftext|>`):
+        tokenizer_file (`str`, *optional*):
+            Path to [tokenizers](https://github.com/huggingface/tokenizers) file (generally has a .json extension) that
+            contains everything needed to load the tokenizer.
```
Comment (ydshieh) [inline src/transformers/models/whisper/tokenization_whisper_fast.py:106]: would love @ArthurZucker to look this part 🙏

**Review by ydshieh — COMMENTED** [review]
A few nits but ready to go 🔥

**Comment by ydshieh** [comment]
Hi @McDonnellJoseph It's fine. No need to dive into those 2 files. I requests 2 tiny changes. Once you commit them, we are good to merge the PR.

**Comment by McDonnellJoseph** [comment]
@ydshieh  No problem I pushed the requested changes :smile:

**Comment by ydshieh** [comment]
Hi, thanks for the commit. May I know why there is irrelevant changes other than pad token id in the last commit pushed?

**Comment by McDonnellJoseph** [comment]
Sorry my linter formatted automatically and I dind't notice it should be fixed now

**Comment by ydshieh** [comment]
We are very close: just need to `make style` and push the change. `tokenization_whisper.py` has some format issue

**Comment by McDonnellJoseph** [comment]
Ok I thing we're good now :smile:

**Review by ydshieh — APPROVED** [review]
Very nice, thank you for the contribution 💯 !

**Pull request merged**